file/configfiles.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+# Get various config files
+
+import logging
+import os
+
+from glob import glob
+
+from file.base import FileCollecting
+from utils import util
+from utils import fileopt
+
+
+class InsightConfigFiles(FileCollecting):
+    def save_sysconf(self):
+        cmd = ["sysctl", "-a"]
+        path_limit_file = "/etc/security/limits.conf"
+
+        # save limits.conf
+        self.save_to_dir(srcfile=path_limit_file)
+
+        # save `sysctl -a`
+        stdout, stderr = util.run_cmd(cmd)
+        if stdout:
+            fileopt.write_file(os.path.join(
+                self.outdir, "sysctl.conf"), stdout)
+        if stderr:
```

Comment:
did we need raise exception

Reply:
It's ok if certain collecting task fails, and it might be not really failing when `stderr` is not `None`. I'll change this to a warning log instead of write to file.